tienvx/mbt-bundle/1.15/config/packages/tienvx_mbt.yaml

```diff
@@ -0,0 +1,18 @@
+imports:
+    - { resource: dev/models/*.yaml }
+    - { resource: dev/predefined-cases/*.yaml }
+
+tienvx_mbt:
+#    max_steps: 300
+#    transition_coverage: 100
+#    place_coverage: 100
+#    default_bug_title: 'New bug'
```

Comment:
No need to have config in comments like this. Rather give a link to your documentation.

Reply:
I simply remove the comments.